apps/services/routers.py

```diff
@@ -262,6 +263,29 @@ async def read_request_details(
     }
 
 
+@services_router.post("/{university_id}/create_students/",
```

Comment:
change create_students to create-students
Don't use "_" in url

Reply:
ok